docs/data-process.rst

```diff
@@ -355,6 +355,47 @@ YP_TIME_REF=whatever``, then you will get the value ``whatever`` instead of the
 reference time.
 
 
+Cast Value Variable Substitution
+--------------------------------
+
+For non-string scalar values, i.e. integers, floats and booleans, the YAML
+processor utility supports casting the value to the correct type before using
+it for substitution:
```

Comment:
Might be worth explaining at this point **why** this is useful.

Reply:
Done.